src/java.desktop/share/classes/java/awt/Component.java

```diff
@@ -1003,7 +1003,7 @@ void initializeFocusTraversalKeys() {
      * @see    #setName
      * @since 1.1
      */
-    public String getName() {
+    public @Nullable String getName() {
```

Comment:
I could see also adding `@Nullable` on the parameter of `setName`, but I didn't give it much thought.